install/windows/build.py

```diff
@@ -41,12 +41,10 @@ def add_size_to_version():
 
 def build_odemisviewer_exe():
     run_command(cpy_command)
-    return run_command(pyi_command, "odemis")
```

Comment:
You still need to call `run_command(pyi_command, "odemis")`. What I meant was that you drop the `return`, but not the rest of the line ;-)

Reply:
ups :D